tests/json-fixtures/test_state.py

```diff
@@ -295,6 +302,8 @@ def test_state_fixtures(fixture, fixture_vm_class):
         computation, _ = vm.apply_transaction(transaction)
     except ValidationError as err:
         transaction_error = err
+        LOGGER.warn("Got transaction error:")
```

Comment:
Was this left in on purpose.  I'm not sure that these logging statements are needed and I'm worried that they could cause confusion since errors here are expected and part of the tests.

At minimum, I'd like a comment added explaining as much, but I also think simply reverting this change should be fine.

Reply:
This went into master in #327 (https://github.com/ethereum/py-evm/pull/327/files#diff-412113be18fef671d701f02096a24e07). I found this invaluable when I was testing, since tests (which were supposed to pass) were failing, and the exceptions were being swallowed silently. I agree this is tricky. Maybe the thing to do here is to switch this from `warn` to `info` or `debug` and add a comment? We should probably do the same to master.